pkgs/servers/web-apps/wiki-js/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, lib, nixosTests }:
+
+stdenv.mkDerivation rec {
+  pname = "wiki-js";
+  version = "2.5.191";
+
+  src = fetchurl {
+    url = "https://github.com/Requarks/wiki/releases/download/${version}/${pname}.tar.gz";
+    sha256 = "sha256-lEHelZTFZxcds7t3TCMcd9b3rKdml54A0/V7gcQIyPA=";
+  };
+
+  buildCommand = ''
+    mkdir $out
+    tar xzvf $src -C $out
+  '';
+
+  passthru.tests = { inherit (nixosTests) wiki-js; };
+
+  meta = with lib; {
+    homepage = "https://js.wiki/";
+    description = "A modern and powerful wiki app built on Node.js";
+    license = licenses.agpl3;
```

Comment:
agpl3Plus or agpl3Only?

Reply:
What exactly is the difference? This not very helpful "or later" clause from spdx.org seems to mean that later versions of agpl3 though AFAIU there's only a agpl3 as latest version.

Reply:
The long term plan is to remove ``licenses.agpl3`` because it is not clear if it is agpl3Only or agpl3Plus. You are right there is no agpl4 but there might be eventually be one or a gpl31. This is more important for agpl21 or less.

Reply:
@SuperSandro2000 yeah, I think I read about this somewhere. Just wondering how to check this for a package like this in question. But from your reasoning I assume that it's best to use agpl3Plus then?

Reply:
The license text contains ``or at your option any later version`` if it is plus.

According to there headers it would be agpl3Only because they do not mention or any later version. https://github.com/Requarks/wiki/blob/dev/server/index.js#L3

Reply:
I see, thanks for the explanation! Pushed a fix :)